pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1791,6 +1792,26 @@ Question.Decimal:
           type: array
           items:
             $ref: '#/Answer.Decimal'
+Question.Equation:
+  allOf:
+    - $ref: '#/Question'
+    - type: object
+      required:
+        - answers
+      properties:
+        expression:
+          type: string
+          description: the expression that must be computed.
+        scale:
+          type: integer
+          description: the number of digits allowed to the right of the decimal point.
+          minimum: 0
+          default: 0
+        readonly: true
```

Comment:
This is not valid OpenAPI- a [Schema][schema] object is required here.


[schema]: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#schema-object

Reply:
Hm... Will check

Reply:
What was the goal you were trying to reach here?

Reply:
JSON Schema and, by extension, OpenAPI, does not support specifying fixed values in protocol responses. I'd recommend making sure `readonly` is always set from the server's side, and making a note about this behavior in the `description` for the `Question.Equation` schema. Unfortunately, due to how our API is structured, this information won't be easily viewable in the rendered docs at the moment.

Reply:
So, as far as I understand, we don't need readonly at all here. So I removed it